testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml

```diff
@@ -20,6 +20,8 @@ metadata:
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
+  cleanup:
```

Comment:
Like I said in #4693, this field needs to be removed from file contents before they are written to disk. This field doesn’t do anything, and I don’t want users to think it is a new feature.

Reply:
Done.